### PR TITLE
PRO-1281: Pillar Wallet - One time warning screen

### PR DESCRIPTION
**Pull request by lalitkumarrock**
https://linear.app/pillarproject/issue/PRO-1281/pillar-wallet-one-time-warning-screen

## Description

- Fixed the emoji alignment on the one time warning screen
- Fixed the fee calculation behaviour on the one time warning screen

## How Has This Been Tested?

- Tested on Simulator 12 Pro Max

## Screenshots (if appropriate):
https://user-images.githubusercontent.com/82652040/230622394-939d10d9-802b-447b-8fc6-ce4465056ab4.mp4

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/screens/Home/ReceiveTokensWarning.tsx [hunk 1]
```diff
@@ -99,7 +99,7 @@ const ReceiveTokensWarning: FC<IReceiveTokensWarning> = ({ theme }) => {
       if (!chainRates || !gasInfo?.gasPrice?.fast) return null;
       return calculateDeploymentFee(chain, chainRates, fiatCurrency, gasInfo);
     },
-    [chainRatesPerChain, chainRatesPerChain, fiatCurrency],
+    [chainRatesPerChain, chainRatesPerChain, fiatCurrency, gasInfoPerChain],
```
Comment (IAmKio) [inline src/screens/Home/ReceiveTokensWarning.tsx:102]: Was the issue here that when the gas information became available, we didn't refresh the UI?

**Review by IAmKio — COMMENTED** [review]
Just a question but other than that - looking good!

File: src/screens/Home/ReceiveTokensWarning.tsx (hunk 1 above)
Comment (lalitkumarrock) [reply src/screens/Home/ReceiveTokensWarning.tsx:102]: Yes

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**